gammapy/obs/observation.py

```diff
@@ -85,3 +91,244 @@ def select_linspace_subset(self, num):
         # Round down to nearest integer
         indices = indices.astype('int')
         return self[indices]
+
+    def select_obs_var_range(self, selection_variable,
+                             value_min, value_max, inverted=False):
+        """Make an observation table, applying some selection.
+
+        Generic function to apply a 1D box selection (min, max) to a
+        table on any variable that is in the observation table and can
+        be casted into a `~astropy.units.Quantity` object.
+
+        If the inverted flag is activated, the selection is applied to
+        keep all elements outside the selected range.
+
+        Parameters
+        ----------
+        selection_variable : string
+            name of variable to apply a cut (it should exist on the table)
+        value_min : `~astropy.units.Quantity`-like
+            minimum value; type should be consistent with selection_variable
+        value_max : `~astropy.units.Quantity`-like
+            maximum value; type should be consistent with selection_variable
+        inverted : bool, optional
+            invert selection: keep all entries outside the (min, max) range
+
+        Returns
+        -------
+        obs_table : `~gammapy.obs.ObservationTable`
+            observation table with observations passing the selection
+        """
+        obs_table = self
+
+        # check that variable exists on the table
+        if selection_variable not in obs_table.keys():
+            raise KeyError('Key not present in table: {}'.format(selection_variable))
+
+        # read values into a quantity in case units have to be taken into account
+        value = Quantity(obs_table[selection_variable])
+
+        # build and apply mask
+        if not inverted:
+            mask = (value_min < value) & (value < value_max)
+        else:
+            mask = (value_min >= value) | (value >= value_max)
+        obs_table = obs_table[mask]
+        return obs_table
+
+    def select_obs_time_range(self, selection_variable,
```

Comment:
I think `_obs` can be removed from the method name.
This is a method on an `ObservationTable` object, so no need to repeat `obs` in the method name, no?